WordPress/src/main/res/layout/people_list_row.xml

```diff
@@ -1,56 +1,32 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="@dimen/people_list_row_height"
+    android:layout_height="wrap_content"
     android:background="?android:selectableItemBackground">
 
-    <org.wordpress.android.widgets.WPNetworkImageView
-        android:id="@+id/people_list_row_avatar"
-        android:layout_width="@dimen/avatar_sz_medium"
-        android:layout_height="@dimen/avatar_sz_medium"
-        android:layout_marginLeft="@dimen/margin_extra_large"
-        android:layout_marginTop="@dimen/margin_extra_large" />
-
     <org.wordpress.android.widgets.WPTextView
-        android:id="@+id/people_list_row_display_name"
+        android:id="@+id/person_role"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginLeft="@dimen/margin_medium"
-        android:layout_marginRight="@dimen/margin_medium"
-        android:layout_marginTop="@dimen/margin_large"
-        android:layout_toRightOf="@id/people_list_row_avatar"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentRight="true"
+        android:layout_marginEnd="@dimen/margin_extra_large"
+        android:layout_marginRight="@dimen/margin_extra_large"
+        android:layout_marginTop="@dimen/margin_extra_large"
```

Comment:
It seems that the role text could use a bit more top margin to reach the baseline of the person username.